### ISSUE TITLE
Add support for viewing images in compressed archives

### ISSUE THREAD

**Issue by Keith94** [d2phap/ImageGlass#254]
### Steps to reproduce:
1. Add some image files (.jpg/,png) to a compressed archived (.zip/.7z) with 7-Zip file archiver
2. Drag-and-drop your compressed archive onto the ImageGlass window

### Expected behavior:
ImageGlass should load all images inside the archive as if it's a folder

### Actual behavior:
ImageGlass can't open the archive because it's not supported

### System info:
- ImageGlass version: 4.17.26
- OS version: Windows 10

### Other info:
Some image viewers such as Honeyview supports compressed archives by default

**Comment by TheRealMarnes** [comment]
a.k.a. cbr (rar), cbz (zip), cb7 (7z) support

**Comment by Keith94** [comment]
Still nothing huh..

**Comment by shironekomimi** [comment]
I'm trying to use this feature, like I'm trying to open a zip file full of images so that the zip acts like a folder, but I'm afraid I can't do it. Is this feature implemented?

**Comment by leaumar** [comment]
@shironekomimi Nope, as indicated by this ticket still being open. It's wild that last time I commented here was 6 years ago. In the meantime photoqt is still pretty much the only fully functional image viewer I could find.

Edit: xnviewmp also supports it now but only with the right extension (cbz) and through the browser interface

**Comment by ScottRFrost** [comment]
[YACReader](https://yacreader.com) and [NeeView](https://github.com/neelabo/NeeView) are both great for CBZ, but I'd love to see it in ImageGlass because I like the UI better!

**Comment by Ashkar666** [comment]
I would like to vote for this feature as well please.

**Comment by xmha97** [comment]
+1 for this feature! Opening archives directly and viewing the images without extracting would be very useful and improve usability a lot.

**Comment by hcaihao** [comment]
+1 for this feature!
